src/main/scala/tilelink/Broadcast.scala

```diff
@@ -149,12 +319,7 @@ class TLBroadcast(params: TLBroadcastParams)(implicit p: Parameters) extends Laz
       filter.io.update.bits.gaveT := d_what === TRANSFORM_T
       filter.io.update.bits.cacheOH := Mux1H(d_trackerOH, trackers.map(_.cacheOH))
 
-      // Incoming C can be:
-      // ProbeAck     => decrement tracker, drop 
-      // ProbeAckData => decrement tracker, send out A as PutFull(DROP)
-      // ReleaseData  =>                    send out A as PutFull(TRANSFORM)
-      // Release      => send out D as ReleaseAck
```

Comment:
Did this information get lost? Or is it replicated elsewhere now?

Reply:
In has been moved to Top of this file under
`Behavior of [[TLBroadcast]]:`
Actually original comments was wrong or not accurate, since `ProbeAckData` won't decrement tracker(MSHR)
Real behavior should be:

1. ClientA -Acquire-> BH
1. BH -Probe-> ClientB 
1. ClientB -ProbeAckData-> BH
1. BH -Put-> Manger
1. Manger -AccessAckData-> BH

decrement by Acquire should happen in 5. rather than 3.